README.md

```diff
@@ -16,9 +16,24 @@ Pistache was originally created by Mathieu Stefani, but he is no longer actively
 
 For those that prefer IRC over Slack, the rag-tag crew of maintainers idle in `#pistache` on Freenode. Please come and join us!
 
-# Precompiled packages
+# Precompiled Packages
```

Comment:
Should your Debian edits be in a different PR?  What if the raw header logic needs to get rolled back?

Reply:
Sorry, for some reason I thought I had already submitted a PR. I just did as #552.